lib/src/Uncertainty/Algorithm/Sensitivity/SobolIndicesAlgorithmImplementation.cxx

```diff
@@ -43,20 +43,8 @@ static const Factory<SobolIndicesAlgorithmImplementation> Factory_SobolIndicesAl
 /* Default constructor */
 SobolIndicesAlgorithmImplementation::SobolIndicesAlgorithmImplementation()
   : PersistentObject()
-  , inputDesign_()
-  , outputDesign_()
-  , size_(0)
   , bootstrapSize_(ResourceMap::GetAsUnsignedInteger("SobolIndicesAlgorithm-DefaultBootstrapSize"))
   , confidenceLevel_(ResourceMap::GetAsScalar("SobolIndicesAlgorithm-DefaultBootstrapConfidenceLevel"))
-  , referenceVariance_()
-  , varianceI_()
-  , varianceTI_()
-  , aggregatedFirstOrderIndices_()
-  , aggregatedTotalOrderIndices_()
-  , secondOrderIndices_()
-  , firstOrderIndiceDistribution_()
-  , totalOrderIndiceDistribution_()
-  , alreadyComputedIndicesDistribution_(false)
```

Comment:
you don't get nasty warnings about the order of the initializations or something like that by removing all these attributes from the initialization section?

Reply:
no warnings at all here if you dont specify any order, here I initialize only the simple types from the header (that's allowed by C++11)